src/lib/hash/hash.cpp

```diff
@@ -251,10 +251,18 @@ std::unique_ptr<HashFunction> HashFunction::create(const std::string& algo_spec,
       {
       return std::unique_ptr<HashFunction>(new SHAKE_128(req.arg_as_integer(0, 128)));
       }
+   if(req.algo_name() == "SHAKE-128(256)")
```

Comment:
I don't think either of these is necessary (or would ever even be hit at runtime) - we already handle arbitrary SHAKE output lengths with the existing code (that's the `req.arg_as_integer(0, def_val)` which takes the first user param if any), f.ex see the test vector files where `SHAKE-128(1120)` and `SHAKE-256(2000)` are tested